lib/ansible/runner.rb

```diff
@@ -330,6 +331,19 @@ def credentials_info(credentials, base_dir)
         [command_line, env_vars, extra_vars]
       end
 
+      def create_config_file(dir, env)
+        config_file           = File.join(dir, "ansible.cfg")
+        env["ANSIBLE_CONFIG"] = config_file
+
+        File.write(config_file, <<~ANSIBLE_CONFIG)
+          [defaults]
+          local_tmp       = /tmp/.ansible_local_tmp
```

Comment:
do we need to make this thread safe and have different file names?

Reply:
These are directories, so it shouldn't matter.

Reply:
I think sharing the directory is fine, because it was sharing before as well (just happened to be ~/.ansible/<dir>)

Reply:
Oh, I think I miss understood the question.

I figured it made sense to separate these just incase, but I am fine making it one dir if you would prefer.  I can also probably nest it as well, but was just playing it safe incase ansible doesn't do well with `mkdir -p`.

Reply:
No. You heard my question correctly and the answer made sense to me. I have no opinion on remote as no local tmp being the same or different directories